core/trino-main/src/test/java/io/trino/server/TestQueryStateInfoResource.java

```diff
@@ -225,14 +225,11 @@ public void testGetQueryStateInfoDenied()
         }
     }
 
-    @Test(expectedExceptions = UnexpectedResponseException.class, expectedExceptionsMessageRegExp = "Expected response code .*, but was 404")
+    @Test
     public void testGetQueryStateInfoNo()
     {
-        client.execute(
-                prepareGet()
-                        .setUri(server.resolve("/v1/queryState/123"))
-                        .setHeader(TRINO_HEADERS.requestUser(), "unknown")
-                        .build(),
-                createJsonResponseHandler(jsonCodec(QueryStateInfo.class)));
+        assertThatThrownBy(() -> client.execute(prepareGet().setUri(server.resolve("/v1/queryState/123")).setHeader(TRINO_HEADERS.requestUser(), "unknown").build(), createJsonResponseHandler(jsonCodec(QueryStateInfo.class))))
```

Comment:
This line is too long. Let's format like below (same as the above `testGetQueryStateInfoDenied()` method)
```java
        assertThatThrownBy(() -> client.execute(
                prepareGet()
                        .setUri(server.resolve("/v1/queryState/123"))
                        .setHeader(TRINO_HEADERS.requestUser(), "unknown")
                        .build(),
                createJsonResponseHandler(jsonCodec(QueryStateInfo.class))))
                .isInstanceOf(UnexpectedResponseException.class)
                .hasMessageMatching("Expected response code .*, but was 404");
```